app/views/posts/_full_post.html.slim

```diff
@@ -0,0 +1,67 @@
+div.container
+	div.row
+		div.col-md-3
+			.profile_container
+					img class='profilepic_small' src="#{post.user.image}?sz=200"
+					.ticket_name #{post.user}
+		div.col-md-6
+			div.short #{post.short}
+			div.tags 
+				= post.tag_list.map{ |tag| "<span class='tag'>#{ tag }</span>" }.join(", ").html_safe
+			div.timestamp 
+				- difference = time_diff post.updated_at
+				| updated #{difference}
+				= get_follow_text(post.follows)
+
+		div.col-md-3
+			.email_container
+				a href="mailto:#{ post.user.email }"
+					img src=image_path("packet.png")
+					div email me
+
+			.follow_container
+				= link_to follow_post_path(post), remote: true
+					- is_followed = Follow.exists?(post_id: post.id, user_id:current_user.id)
+					- text = is_followed ? 'following' : 'follow'
+					img#following_icon [src=image_path("follow.png")
+										style="display:#{:none if is_followed};"]
+					img#follow_icon  [src=image_path("ok.png")
+									  style="display:#{:none unless is_followed};"]
+					div #{text}
+
+	div.row
+		div.long #{post.long}
+	div.row
+		div.num_comments
+			// See follow text above -- do this in rails helper
+			- if post.comments.any?
+				- if post.comments.count > 1
+					i showing #{post.comments.count} comments
+				- else
+					i showing 1 comment
```

Comment:
you can use the Rails `pluralize` method to convert english words to plural forms based on a provided count.  Rails == wins again.